Juliating

jtingy opened the pull request:
Adding in steps 7-8, filling out Roxygen skeletons for functions, adding lifetable function, also got rid of rows in mortality2 that have na populations.

jtingy committed:
Merge pull request #10 from g-rade/main

Adding new stuff

Commit:
11/20 Adding in final steps and a lifetable function also updated the roxygen skeletons for functions

Commit:
11/20 Fixing stuff for the cmd checks

Commit:
Getting rid of observations with NA population

g-rade commented:
hey just so you know, that sometimes the lag() or lead() creates an NA value, but we can use and if_else() to get that value in the dataset so later calculations are accurate

Commit:
fix merge conflict for pull request

Merge branch 'main' of https://github.com/g-rade/lifetables into juliating

# Conflicts:
#	R/Steps1_3.R
#	R/globals.R
#	data/test_data.rda
